src/components/Profile.js

```diff
@@ -0,0 +1,42 @@
+// @flow
+import React from 'react';
+import styled from 'styled-components';
+import ReactImageFallback from "react-image-fallback";
+import { Tabs } from 'antd';
+
+import { Flex, Box, Island } from './Layout';
+import { borderRadius, borderColor, borderWidth } from 'styled-system'
+import { SmallUser } from '../types/api';
+
+import { colors } from '../util/style';
+import Text from './Text';
+
+const TabPane = Tabs.TabPane;
+const ProfileImage = styled(ReactImageFallback)`
+
+`;
+
+const Profile = ({name, username, profileImageUrl, karma} : SmallUser) => (
+    <Flex align="center" direction="column">
+        <Box my={[1,2,3,3]} borderRadius={100} borderColor={colors.blue} borderWidth={1} width={200} height={200}>
+          <ProfileImage src={profileImageUrl} fallbackImage="http://www.upplanet.com/bighero/outlay-demo/img/customer_avatar.png" alt={username} />
+        </Box>
```

Comment:
I didn't even know you could do this. Did you extend the Box component to support these props?

If not, then yea I don't think props like borderColor, borderWidth, etc... are supported.
If yes, I think it makes sense to add props to the Box component, but these are probably too flexible.

In fact what you probably really want is to make an Image.js component that has these styles by default
```
export default styled(ReactImageFallback)`
...styles that are somewhat configurable.
`
```